### DISCUSSION TITLE
希望能在README中附上App store 跳转链接

### DISCUSSION THREAD

**Discussion by TinyuZhao** [shenruisi/Stay#36] [General]
看到HelloGithub的文章找到这个开源项目的，一直在README里面找安装方式，找了一圈发现没有，最后在[OpenProject](https://openprojectrepo.com/project/shenruisi-Stay-javascript-misc)里找到了安装方式，原来在app store中直接下载就好了😂

**Comment by shenruisi** [comment]
README的大标题就可以直接跳转下载地址哟。

**Reply by TinyuZhao** [reply]
谢谢啦
